util.go

```diff
@@ -117,3 +117,86 @@ func trDelete(s, substr string) string {
 	}
 	return ret.String()
 }
+
+// createMediaMessage returns a `Chattable' (which can be sent with bot.Send)
```

Comment:
Quoting `Chattable' like m4 seems to be confusing godoc (and at least one Go chrome extension I use). Try "Chattable".

Reply:
Done, but we may have quotes like that elsewhere. Maybe we should look into it in another PR.